recipe/meta.yaml

```diff
@@ -10,32 +8,41 @@ package:
 source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  {{ hash_type }}: {{ hash }}
+  sha256: 8a382fe74627f08c0d18908c0713ca4a20aad5385f077579e56208beea2893b2
 
 build:
-  noarch: python
-  number: 1
-  script: python -m pip install --no-deps --ignore-installed .
+  number: 0
```

Comment:
When forking a new feedstock, we should try to keep the build number. Not doing that would cause conda-forge's package to be considered before our package if both defaults and conda-forge are mixed.